Automattic/newspack-plugin
# Unified Components Import

### All Submissions:






### Changes proposed in this Pull Request:

This PR adds a single file with exports for all components to simplify imports, allowing for a single import with destructuring (e.g. `import { Button, ImageUpload, CheckboxControl } from '../../components';` ). This is an interim step before we publish components as an NPM package. Also, changed all component directories to kebab-case to match other Automattic directory structures such as https://github.com/WordPress/gutenberg/tree/master/packages/components/src.

### How to test the changes in this Pull Request:

1. `npm ci && npm run clean && npm run build:webpack`
2. Observe no errors in the build process.
3. Navigate to `/admin.php?page=newspack-subscriptions-wizard`
4. Observe all components appear and function as before.

### Other information:
